src/Sylius/Bundle/PromotionBundle/Resources/translations/messages.en.yml

```diff
@@ -16,6 +16,11 @@ sylius:
             starts_at: Starts at
             usage_limit: Usage limit
         promotion_action:
+            order_fixed_discount: Order fixed discount
+            item_fixed_discount: Item fixed discount
+            order_percentage_discount: Order percentage discount
+            item_percentage_discount: Item percentage discount
+            shipping_percentage_discount: Shipping percentage discount        
```

Comment:
It would be good to keep the key alphabetically sorted. I know that it's not done this way in many places, but if we change something it's good to sort it by the way, just for the specific key ;)